### PR TITLE
GCS:Enable apply/save buttons on vehicle+output pages

### PR DESCRIPTION
**Pull request by tracernz**
Oops. :/ My fail in #1912.

### CODE REVIEW

File: ground/gcs/src/plugins/config/configoutputwidget.cpp [hunk 1]
```diff
@@ -102,18 +102,6 @@ ConfigOutputWidget::ConfigOutputWidget(QWidget *parent)
     connect(m_config->calibrateESC, &QAbstractButton::clicked, this,
             &ConfigOutputWidget::startESCCalibration);
 
-    // Track the ActuatorSettings object
-    addUAVObject("ActuatorSettings");
-
-    // Associate the buttons with their UAVO fields
-    addWidget(m_config->cb_outputRate6);
```
Comment (tracernz) [inline ground/gcs/src/plugins/config/configoutputwidget.cpp:109]: These cause a segfault when ConfigTaskWidget adds the relations itself... ~~and don't actually do anything AFAICT~~ it does make us track their "dirty" state...

File: ground/gcs/src/plugins/config/configoutputwidget.cpp [hunk 2]
```diff
@@ -102,18 +102,6 @@ ConfigOutputWidget::ConfigOutputWidget(QWidget *parent)
     connect(m_config->calibrateESC, &QAbstractButton::clicked, this,
             &ConfigOutputWidget::startESCCalibration);
 
-    // Track the ActuatorSettings object
-    addUAVObject("ActuatorSettings");
```
Comment (tracernz) [inline ground/gcs/src/plugins/config/configoutputwidget.cpp:106]: Well, no point tracking an object unless you actually have a relation to it... which we do anyway.

**Comment by tracernz** [comment]
Checked that the PWM rates and arming checkbox still work as expected.

**Comment by tracernz** [comment]
Keeping some debt around in the interests of getting this blocker fixed. I missed that I still had some work to do in at least output page to really cleanup UAVO relations. :(

**Review by mlyle — APPROVED** [review]

**Comment by tracernz** [comment]
jenkins, test this please

**Comment by mlyle** [comment]
jenkins, test this please

**Pull request merged**